### ISSUE TITLE
Option to disable "Recommended" section stopped working

### ISSUE THREAD

**Issue by vbickov** [valinet/ExplorerPatcher#3788]
It happened after the latest upgrade 23H2. The option to disable "Recommended" in the Start Menu section does not have any effect.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3313

please close this.

**Comment by vbickov** [comment]
It's up to you. But the case you mentioned was submitted on May 24. Update 23H2 was released a week ago. It does not "fail occasionally." It simply does not work.

**Comment by pyrates999** [comment]
Yes, that's how long this has not been working for.

**Comment by vbickov** [comment]
Sorry, but this does not seem correct as well. I'm using it every day, and it stopped working after the update 23H2. My point is even though the symptoms are the same, the issue may be different.

**Comment by pyrates999** [comment]
You need to prove it then.

**Comment by vbickov** [comment]
Are you saying it works as expected with 23H2 on your side? Before I applied the 23H2 update, everything was fine. Why do I need to prove anything? Do you think I have nothing else to do except report unexistent issues?

**Comment by vbickov** [comment]
For those who are still struggling, In my case, this had something to do with display scaling. I found clues [here](https://answers.microsoft.com/en-us/windows/forum/all/remove-recommended-section-from-start-menu-is-not/ba6c469e-565b-4eb1-9d75-12ba0dcc75c3?page=1):

The corresponding group policy and other settings they recommended did not have any effect until I remembered that I had slightly changed the display scale. I usually use a custom scale of 195% because 200% makes the mouse cursor too big. After applying the 23H2 update, my custom scale settings somehow got reset, and for some weird reason, I applied 198%. It turns out a 3% difference is enough for the "Recommended" section to reappear.  Reducing the scale to 195% resolved the issue. 

This is not a fluke, and I confirmed this several times. Visually, a 3% difference does not affect the size of icons. But, maybe it has something to do with how MS calculates the number of icon rows it can show before the "Recommended" section.
